frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/SafeguardForm/SafeguardForm.tsx

```diff
@@ -265,16 +266,19 @@ export const SafeguardForm = ({
         <StyledFormContainer onSubmit={handleSubmit} mode={mode}>
             <StyledTopRow sx={{ mb: 1 }}>
                 <StyledIcon />
-                <StyledLabel>Pause automation when</StyledLabel>
+                <StyledLabel sx={{ mr: 'auto' }}>
+                    Pause automation when
+                </StyledLabel>
                 {mode !== 'create' && onDelete && (
-                    <IconButton
+                    <PermissionIconButton
+                        permission={UPDATE_FEATURE_STRATEGY}
                         onClick={handleDelete}
                         size='small'
                         aria-label='Delete safeguard'
-                        sx={{ padding: 0.5, marginLeft: 'auto' }}
```

Comment:
sx on permission button is not applied to the outermost element breaking margin so I moved to previous element StyledLabel